packages/augur-ui/src/modules/common/icons.tsx

```diff
@@ -21,6 +21,58 @@ export const AugurLoadingLogo = (
   </svg>
 );
 
+export const Sports = (
```

Comment:
These icons need to be cleaned up a little bit, remove the extra attributes like xmlns. Also fix all the `-` to use camel case instead as our linter will eventually not like this. so `stroke-width` => `strokeWidth`